src/parser/jobs/smn/modules/Aetherflow.js

```diff
@@ -135,6 +147,22 @@ export default class Aetherflow extends Module {
 			tiers: PAINFLARE_SEVERITY,
 			value: numBadPainflares,
 		}))
+
+		// Energy Siphon suggestion, I want to say < 3 is medium because 3 on two targets is greater than a full Energy Drain (or 2 casts on 1 target)
+		const numBadEnergySiphons = this._badEnergySiphons.length
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.ENERGY_SIPHON.icon,
+			content: <Trans id="smn.aetherflow.suggestions.energysiphon.content">
+				Avoid casting <ActionLink {...ACTIONS.ENERGY_SIPHON}/> on less than three targets, as it deals less damage than <ActionLink {...ACTIONS.ENERGY_DRAIN}/> per hit.
+			</Trans>,
+			why: <Trans id="smn.aetherflow.suggestions.energysiphon.why">
+				{numBadEnergySiphons} single- or two-target
```

Comment:
wording nit; would probably flow better to say "X casts? of ES hit less than 3 targets"

also needs updating to say ES instead of PF

Reply:
Agreed.

As you may have guessed from the end of the why string saying Painflare, I just copied the Painflare message and tweaked it to account for the 3 target minimum instead of 2.